system_setup/client/client.py

```diff
@@ -36,21 +37,43 @@ class SystemSetupClient(SubiquityClient):
         "WSLIdentity",
         "WSLConfigurationBase",
         "Summary",
+    ]
+
+    variant_to_controllers = {
+        "wsl_setup": controllers,
+        "wsl_configuration": [
+            "WSLConfigurationBase",
+            "WSLConfigurationAdvanced",
+            "Summary",
         ]
+    }
+
+    def restart(self, remove_last_screen=True, restart_server=False):
```

Comment:
Wait, didn't I ask you to parameterize the superclass implementation of this method in a previous PR?

Reply:
I am not clear about this.... could you elaborate more?

Reply:
Having read things a bit more, I don't think you need to add the restart method to this class at all. Can you explain why you did that?

Reply:
Now I check again, yep you are correct, the restart is not needed here.

Reply:
done for the fixes here.